src/helpers/web3.js

```diff
@@ -11,6 +11,13 @@ export const web3Instance = new Web3(
   new Web3.providers.HttpProvider(`https://mainnet.infura.io/`)
 );
 
+/**
+ * @desc web3 ropsten http instance
+ */
+export const web3RopstenInstance = new Web3(
+  new Web3.providers.HttpProvider(`https://ropsten.infura.io/`)
+);
```

Comment:
Just use the web3SetProvider to change to `ropsten` or even more simply just change the default instance to `ropsten`

Reply:
Ok cool, I'll change the default instance to ropsten